docs/content/about-iota/iota-wallet-dashboard/how-to/vesting.mdx

```diff
@@ -0,0 +1,50 @@
+---
+description: A guide on how to use the IOTA Wallet Dashboard dApp to view, claim and stake vested tokens.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet-dashboard]
+---
+
+# Manage Vesting and Time-locked Assets
+
+You can get overview of your vested tokens in the Dashboard's home view, as well as access the staking feature for your
+currently vested tokens.
+
+## View Currently Vested tokens
+
+You can access the Dashboard's Vesting view by clicking on the `Vesting` button on the left side menu.
+
+   ![Click Vesting](/img/about-iota/iota-wallet-dashboard/how-to/vesting/access-vesting.png)
+
+### See Vesting Schedule
+
+You can get a detailed view of your rewards schedule by clicking the `See All` button.
+
+   ![Vesting Home](/img/about-iota/iota-wallet-dashboard/how-to/vesting/vesting-home.png)
+
+This will open a vesting calendar with details on your rewards schedule.
+
+   ![Rewards Schedule](/img/about-iota/iota-wallet-dashboard/how-to/vesting/rewards-schedule.png)
+
+## Claim Vesting Rewards
+
+You can claim your currently unlocked vesting rewards by clicking the `Collect` button, and then confirming the
+transaction in your wallet.
+
+   ![Vesting Home](/img/about-iota/iota-wallet-dashboard/how-to/vesting/vesting-home.png)
+
+## Stake Vested Tokens
+
+### From the Dashboard's Home View
```

Comment:
This section is incorrect, you cant stake vested tokens from the staking view, you can only do it from the vesting view. The staking tab is for normal staking only, not timelocks